news/atom.xml

```diff
@@ -14,8 +14,11 @@ layout:
     <name>The ev3dev team</name>
   </author>
 
+  {% assign index = 0 %}
```

Comment:
I missed that part. Maybe better to call it `count` instead of `index` in that case.